roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -175,6 +175,14 @@
   notify: Restart crio
   when: http_proxy is defined or https_proxy is defined
 
+- name: Cri-o | write cri-o options drop-in
+  copy:
+    src: options.conf
+    dest: /etc/systemd/system/crio.service.d/options.conf
+    mode: 0644
+  notify: Restart crio
+  when: (container_checkpoint_enabled is defined) and container_checkpoint_enabled
```

Comment:
Using  `container_checkpoint_enabled | default(false)` can make it more readable